.gitignore

```diff
@@ -5,3 +5,7 @@
 /.synapse-base
 /synapse
 /var
+*~
+\#*
+logs/
```

Comment:
can we make these `/logs` and `/results.tap` rather than matching those filenames anywhere?